### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by lukman155**
This is my submission link for my React/Redux Capstone project

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const NavBar = () => (
+  <nav>
+    <Link to="/">
+      PLAYFREE
+    </Link>
+    <div>
+      <button type="button">Categories</button>
+    </div>
+  </nav>
```
Comment (Munsa1) [inline src/components/NavBar.js]: - [ ] It is a requirement that the Home page has a list of categories that could be filtered using given parameters, e.g. category name. kindly implement this.
image

File: src/components/Homepage.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <>
+      <NavBar />
+      <section>
+        <div>
+          <p className="game-title">
+            <b>371</b>
+            {' '}
+            free-to-play games found in our games list!
+
+          </p>
```
Comment (Munsa1) [inline src/components/Homepage.js]: - [ ] Kindly follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

* Typography: font face, size, and weight
* Layout: composition and space between elements.follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

Typography: font face, size, and weight
Layout: composition and space between elements.

Given Design
image
image

Your Design
image

#### Note
You can select one main colour for the app and be consistent (it does **not** need to be the same colour as in the design guidelines).

File: src/__test__/navbar.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+describe('Testing Card page component', () => {
+  it('should render correctly Card page component', () => {
+    const tree = render(
+      <BrowserRouter>
+        <NavBar />
+      </BrowserRouter>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (Munsa1) [inline src/__test__/navbar.test.js:15]: - [ ] Kindly add some more unit tests for several components of the app and some pure functions in Redux code.

Comment (Munsa1) [inline src/__test__/navbar.test.js:15]: - [ ] Kindly ensure the App has integration tests using React Testing Library

* you can perform a mocking data fetching and Redux Store connection.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @lukman155 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @lukman155, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**